src/styles/typography/Body/Body2.tsx

```diff
@@ -0,0 +1,29 @@
+import { colors } from "../../colors";
+import { typography } from "../typography";
+
+type Body2Props = {
```

Comment:
nit: poderiia ser um interface tb :)

```suggestion
interface Body2Props {
```